src/debiai/debiai.js

```diff
@@ -193,3 +193,15 @@ exports.selectionDataIdList = (req, res) => {
     }
 }
 
+exports.createSelection = (req, res) => {
```

Comment:
Please add comment to the last two exported functions (selectionList and selectionDataIdList) explaining what the expected json format is.